src/commands/zork.js

```diff
@@ -0,0 +1,39 @@
+const BaseCommand = require('../base-commands/base-command');
+
+class Zork extends BaseCommand {
+  async run() {
+    await super.run();
+
+    let launchZork = null;
+    try {
+      launchZork = require('zorkjs');
+    } catch (error) {
+      // We'll need to load the zorkjs module
+    }
+
+    if (!launchZork) {
+      this.logger.warn('Standby, loading the dungeon...');
+      const exec = require('await-exec');
+      await exec('npm install zorkjs');
+
+      try {
+        launchZork = require('zorkjs');
+      } catch (error) {
+        // I guess it didn't work :(
+      }
+    }
+
+    if (launchZork) {
+      launchZork();
+    } else {
+      this.logger.error('I don\'t know the word "zork".');
```

Comment:
If we can't get the zorkjs npm package loaded for whatever reason, we still pay a little homage using the phrasing from the game when you type a command it doesn't understand.

Reply:
the error should probably be loaded into the catch of line 27 instead of here. If a try/catch succeeds and you are only doing a truthy check on `launchZork` properly implemented there is not really much way for that lookup to fail at this point